src/main/java/me/mrCookieSlime/Slimefun/SlimefunPlugin.java

```diff
@@ -1,11 +1,7 @@
 package me.mrCookieSlime.Slimefun;
 
 import java.io.File;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.List;
-import java.util.Set;
+import java.util.*;
```

Comment:
Please do not use the star import.

Reply:
Yeah. I'm trying to find a solution for that :P 
My IntelliJ IDEA does that automatically. I guess I'll have to set them manually, as well as add new imports for entries.